File: CivitasJava/src/civitas/JuegoPrueba.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

import juegoTexto.Controlador;
import juegoTexto.VistaTextual;
        
import java.util.ArrayList;
import java.util.Arrays;

/**
 *
 * @author luis
 */
public class JuegoPrueba {
    
    public static void main(String[] args){
       ArrayList<String> jugadores = new ArrayList<>(Arrays.asList("Luis", "Fran", "Miguel")); 
       Dado dado = Dado.getInstance();
       dado.setDebug(Boolean.TRUE);
       
       CivitasJuego nuevo = new CivitasJuego(jugadores);
       VistaTextual vista = new VistaTextual();
       Controlador control = new Controlador(nuevo,vista);
       control.juega();
       
    }
}

File: CivitasRuby/lib/Juego_prueba.rb
# encoding:utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
require_relative 'dado.rb'
require_relative 'civitas_juego.rb'
require_relative 'tablero.rb'
require_relative 'jugador.rb'

module Civitas
  class Juego_prueba
    def initialize
      
    end
    
    def self.main
      
      jugadores = Array.new
      jugadores << "Luis"
      jugadores << "Fran"
      jugadores << "Miguel"
      
      dado = Dado.instance
      
      juego = CivitasJuego.new(jugadores)
      vista = Vista_textual.new
      controlador = Controlador.new(juego, vista)
      controlador.juega
    end
  end

  
  Juego_prueba.main
end
